content/influxdb/cloud-dedicated/process-data/downsample/downsample-quix.md

```diff
@@ -32,7 +32,7 @@ This guide also assumes you have already
 
 The following diagram illustrates how data is passed between processes as it is downsampled:
 
-{{< html-diagram/quix-downsample-pipeline >}}
+{{< html-diagram/influxdb-v3-quix-downsample-pipeline >}}
 
 {{% note %}}
 It is usually more efficient to write raw data directly to Kafka rather than
```

Comment:
Further down on line 68, dependencies list should match what's actually pip installed.

Reply:
@tbedford not sure what you're referring to here